### PR TITLE
Validation on additional signin fields in signin form

### PR DESCRIPTION
**Pull request by kashif938**
**Description:** 
Added validation on all additional fields in signin form.

**JIRA:** 
https://edlyio.atlassian.net/browse/EDE-758

**Merge checklist:**

- [ ] All reviewers approved
- [ ] CI build is green
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are (reasonably) squashed
- [ ] Translations are updated

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: openedx/core/djangoapps/user_api/accounts/__init__.py [hunk 1]
```diff
@@ -66,11 +82,29 @@
     _(u"Enter a valid email address that contains at least {min} characters."),
     min=EMAIL_MIN_LENGTH,
 )
+SCH_ORG_BAD_LENGTH_MSG = format_lazy(
+    _(u"School / Organization name must be between {min} and {max} characters long."),
+    min=SCH_ORG_MIN_LENGTH,
+    max=SCH_ORG_MAX_LENGTH
+)
+PHONE_BAD_LENGTH_MSG = format_lazy(
+    _(u"Phone must be between {min} and {max} characters long."),
```
Comment (danialmalik) [inline openedx/core/djangoapps/user_api/accounts/__init__.py:91]:
```suggestion
    _(u"Phone number must be between {min} and {max} characters long."),
```

**Review by danialmalik — APPROVED** [review]
LGTM

File: lms/static/js/student_account/views/FormView.js [hunk 2]
```diff
@@ -17,7 +17,13 @@
             formErrorsTpl: formErrorsTpl,
             formErrorsJsHook: 'js-form-errors',
             defaultFormErrorsTitle: gettext('An error occurred.'),
-            events: {},
+            events: {
```
Comment (HamzaIbnFarooq) [inline lms/static/js/student_account/views/FormView.js:20]: Kindly use [REDHOUSE_CUSTOM] comment whenever you do any change in edx's core code.

File: openedx/core/djangoapps/user_api/accounts/__init__.py (hunk 1 above)
Comment (HamzaIbnFarooq) [reply openedx/core/djangoapps/user_api/accounts/__init__.py:91]:
```suggestion
    _(u"Phone number must range between {min} and {max} characters."),
```

File: openedx/core/djangoapps/user_api/validation/views.py [hunk 3]
```diff
@@ -15,7 +15,10 @@
     get_name_validation_error,
     get_password_validation_error,
     get_username_validation_error,
-    get_username_existence_validation_error
+    get_username_existence_validation_error,
```
Comment (HamzaIbnFarooq) [inline openedx/core/djangoapps/user_api/validation/views.py:18]: kindly sort these imports.

**Review by HamzaIbnFarooq — COMMENTED** [review]
Some small changes.

**Review by danialmalik — APPROVED** [review]

**Review by HamzaIbnFarooq — APPROVED** [review]

**Pull request merged**